packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -132,4 +132,60 @@ The `Button.Group` can be displayed vertically using the `vertical` prop.
 `Button.Group` can also be used to create a split button. <br />
 The dropdown can change the button action (as in this example) or be used to immediately trigger a related action.
 
-<Story id="inputs-button--group-split" />
\ No newline at end of file
+<Story id="inputs-button--group-split" />
+
+## Button.Toggle
+Is used to group related **options**.
+
+### Usage
+
+The `Button.Toggle`
+<ul>
+  <li>emphasizes groups of related <code>Toggle</code> buttons,</li>
+  <li>controls the selected state of its child buttons when given its own <code>selectedIndexes</code> prop.</li>
+</ul>
+
+```tsx
+import { Button } from '@equinor/eds-core-react'
+import { save } from '@equinor/eds-icons'
+
+<Button.Toggle aria-label="actions">
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+</Button.Toggle>
+```
+
+### Accessibility
+
+#### ARIA
+<ul>
+  <li>Button.Toggle sets <code>aria-pressed="bool"</code> according to the button state.</li>
+  <li>You should provide an accessible label with <code>aria-label="label"</code>, <code>aria-labelledby="id"</code> or <code>&lt;label&gt;</code>.</li>
+</ul>
+
+## Examples
+
+### Exclusive selection
```

Comment:
Maybe call it "single selection" instead of "exclusive selection" so the word is similar to our other components such as `Autocomplete`& `NativeSelect`? 🤔

Reply:
Definitely, especially when we have already `Multiple selection`.